modules/tinymce/src/core/main/ts/content/InsertContentImpl.ts

```diff
@@ -58,17 +58,62 @@ const trimBrsFromTableCell = (dom: DOMUtils, elm: Element) => {
   Optional.from(dom.getParent(elm, 'td,th')).map(SugarElement.fromDom).each(PaddingBr.trimBlockTrailingBr);
 };
 
+const hasInheritableStyles = (styles: string[]): boolean => {
+  // TINY-7326: Figure out what else should go in the nonInheritableStyles list
+  // Elements which have these styles should not get merged as they have non-inherited properties
+  const nonInheritableStyles = [ 'margin', 'padding', 'border', 'background' ];
```

Comment:
https://dev.to/muyunyun/inherited-and-non-inherited-in-css-mi4#noninherited-type is something I found quickly that would let us get a better list for this.

- [ ] Consider adding more to this list

Reply:
Thanks. Have added those other ones to the list. Although, I am not 100% certain on the best way to efficiently lookup all of the strings.